### PR TITLE
HDDS-3973. Update main feature design status.

### PR DESCRIPTION
**Pull request by ChenSammi**
https://issues.apache.org/jira/browse/HDDS-3973

### CODE REVIEW

**Comment by ChenSammi** [comment]
@nandakumar131  Could you take a look at this JIRA, make sure I have your name correctly spelled.

File: hadoop-hdds/docs/content/design/decommissioning.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ title: Decommissioning in Ozone
 summary: Formal process to shut down machines in a safe way after the required replications.
 date: 2019-07-31
 jira: HDDS-1881
-status: implementing
+status: implemented
```
Comment (elek) [inline hadoop-hdds/docs/content/design/decommissioning.md:6]: It's more like abandoned. The 90 % of the features are implemented but they are not merged back to the master and it's not actively rebased.

Comment (ChenSammi) [reply hadoop-hdds/docs/content/design/decommissioning.md:6]: @elek , what do you mean by "abandoned",  do we plan to resume the implementation later?  It seems Ozone so far dosn't support decommission a datanode. But we definitely need this capability.

Comment (elek) [reply hadoop-hdds/docs/content/design/decommissioning.md:6]: Ok. The abandoned was a wrong word. It's in *implementing* state, but as far as I know, nobody is working actively on it  (Is it correct, @sodonnel?). We need to invest a few more days/weeks to finish it. (Design is discussed multiple times, and we had an agreement about the implementation).

Comment (sodonnel) [reply hadoop-hdds/docs/content/design/decommissioning.md:6]: Yes - at the moment nobody is actively working on decommission. We do need to pick it back up at some point soon, as its an important feature. The branch was last rebased some time ago. I fear rebasing it now will be tricky as a lot has changed.

Comment (ChenSammi) [reply hadoop-hdds/docs/content/design/decommissioning.md]: I see. Thanks for the info.  Let me know once you guys resume the implementation. I'm happy to provide any help. It's a must-to-have feature for us to smoothly maintain the production cluster.

**Pull request merged**